troi/patches/playlist_from_listenbrainz.py

```diff
@@ -34,5 +38,24 @@ def description():
         return "Retrieve a playlist from the ListenBrainz"
 
     def create(self, inputs):
+
+        mbid = inputs["mbid"]
+        jspf = inputs["jspf"]
+    
+        if mbid == "":
+            mbid = None
+        if jspf == "":
+            jspf = None
+
+        if mbid is not None and jspf is not None:
+            raise RuntimeError("Cannot pass both playlist mbid and jspf to TransferPlaylistPatch.")
+
+        if mbid is None and jspf is None:
+            raise RuntimeError("Must pass either playlist mbid or jspf to TransferPlaylistPatch.")
+
+        if jspf is not None:
```

Comment:
You can remove this check because `isinstance(None, str)` is false.

Reply:
Fixed.